### ISSUE TITLE
Basic question on fine tuning...my results are not even close...

### ISSUE THREAD

**Issue by mikedorin** [xinjli/allosaurus#84]
I have a small dataset. I have about 100 words in the train directory and 31 words in the validate directory.
I run the steps for a fine-tune with no issues.
It runs for 90 epochs then stops, the output is below.

I run:
python -m allosaurus.run --lang bura1267 --model=mad999 --device_id=1 -i train/index_7_botella.wav 
I sort of expect to see something in the ballpark of : ah ak al ad aq ar

instead, I see:
ag ab

I have updated bura1267 with my own list of phones, by the way. 

Do I need more data? Do I need to prep my wave files better?   Any suggestions?
Thank you,
 Mike

training results:
saving model
epoch[batch]: 88[0000] | train loss 3.06233 train per 0.78426
epoch88 | validate per : 0.94444
epoch[batch]: 89[0000] | train loss 3.04346 train per 0.80758
epoch89 | validate per : 0.93056
epoch[batch]: 90[0000] | train loss 3.02284 train per 0.79883
epoch90 | validate per : 0.93750
no improvements for several epochs, early stopping now
